modules/apps/content-dashboard/content-dashboard-web/build.gradle

```diff
@@ -21,15 +21,13 @@ dependencies {
 	compileOnly project(":apps:asset:asset-categories-api")
 	compileOnly project(":apps:asset:asset-display-page-api")
```

Comment:
@cgoncas, was testing, and we can remove  compileOnly project(":apps:asset:asset-display-page-api") and ompileOnly project(":apps:layout:layout-display-page-api") too